Sync/BookmarkPayload.swift

```diff
@@ -352,7 +352,7 @@ open class BookmarkPayload: BookmarkBasePayload {
     }()
 
     lazy var tagsString: String = {
-        if self["tags"].type == Type.array {
+        if self["tags"].isArray() {
```

Comment:
I was reading the SwiftyJSON source code as I adjusted these, because I was also concerned.

All `JSON` instances have a dictionary and an array member; having the right type code is the only thing we can check there, because you'll never get a `nil` array if the type code is `.array`:

```
    //Optional [JSON]
    public var array: [JSON]? {
        if self.type == .array {
            return self.rawArray.map { JSON($0) }
        } else {
            return nil
        }
}

…

fileprivate var rawArray: [Any] = []
```